plugins/ui/src/js/src/elements/IllustratedMessage.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {
+  IllustratedMessage as DHCAIllustratedMessage,
+  IllustratedMessageProps as DHCAIllustratedMessageProps,
+  Icon,
+} from '@deephaven/components';
+import { isElementOfType } from '@deephaven/react-hooks';
+
+export function IllustratedMessage(
+  props: DHCAIllustratedMessageProps
+): JSX.Element {
+  const { children, ...otherProps } = props;
+
+  /* eslint-disable-next-line react/jsx-props-no-spreading */
+  if (children === undefined) return <DHCAIllustratedMessage {...props} />;
+
+  const newChildren = React.Children.map(children, element => {
+    if (isElementOfType(element, Icon) === true) {
+      const size = element.props.size ?? 'XL';
```

Comment:
Size should be XXL, not XL
```suggestion
      const size = element.props.size ?? 'XXL';
```